### ISSUE TITLE
Rephrase pinned messages "open" button

### ISSUE THREAD

**Issue by notramo** [ajbura/cinny#2106]
### Describe the problem

1. Send a link in a message (and nothing else just a link)
2. Pin the message
3. Click on pinned messages in Cinny: "open" implies that it will open the link

### Describe the solution you'd like

A better label for the button would be "scroll to message", "jump" or "jump to message"

**Comment by kfiven** [comment]
I disagree, open is much better terminology as that what the button do, "open the message into timeline". Open is also used in message search results and notifications to do the same thing. "scroll to message" or "jump to message" are too long for the button, "jump" is good option but I think we should stick with "open" as users are already familiar with it.

**Comment by ajbura** [comment]
closing as ^
